README.md

```diff
@@ -38,6 +38,12 @@ $ mvn clean install
 
 The build creates artifacts which will be used in the rest of this guide.
 
+ To generate code coverage metrics, execute the following command:
```

Comment:
To generate code coverage metrics, execute the following command: -> To generate "aggregate" code coverage metrics, execute the following command: